spyrmsd/graph.py

```diff
@@ -1,42 +1,102 @@
+from spyrmsd import constants
+
+import numpy as np
+import warnings
+
+_available_backends = []
+_current_backend = None
+
+## Backend aliases
+_graph_tool_aliases = ["graph_tool", "graphtool", "graph-tool", "graph tool", "gt"]
+_networkx_aliases = ["networkx", "nx"]
+
+## Construct the alias dictionary
+_alias_backendDict = {}
```

Comment:
```suggestion
_alias_to_backend = {}
```
Maybe? Not sure, but I'm not fond of mixing snake case and camel case on a single variable.